test/e2e/registry_test.go

```diff
@@ -0,0 +1,174 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
+
+	"github.com/knative/eventing/test"
+	pkgTest "github.com/knative/pkg/test"
+	"k8s.io/apimachinery/pkg/util/uuid"
+)
+
+// Helper to setup test data and expectations.
+type testFixture struct {
+	policy             *v1alpha1.BrokerPolicySpec
+	preRegisterEvent   bool
+	wantEventDelivered bool
+}
+
+func TestRegistryBrokerAllowAnyAccept(t *testing.T) {
```

Comment:
Currently for the new e2e tests, you'll need to add them in the `channelTestMap` of `main_test.go`, otherwise they will not be run. We are trying to figure out a better way to manage them though.

Reply:
Added, thanks!